extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml

```diff
@@ -336,31 +337,31 @@
 	</channel-type>
 
 	<channel-type id="age">
-		<item-type>Number</item-type>
+		<item-type>Number:Time</item-type>
 		<label>Moon age</label>
 		<description>The age of the moon in days</description>
-		<state readOnly="true" pattern="%d d" />
+		<state readOnly="true" pattern="%d %unit%" />
 	</channel-type>
 
 	<channel-type id="ageDegree">
-		<item-type>Number</item-type>
+		<item-type>Number:Angle</item-type>
 		<label>Moon age</label>
-		<description>The age of the moon in degree</description>
-		<state readOnly="true" pattern="%.2f °" />
+		<description>The age of the moon</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
 	</channel-type>
 
 	<channel-type id="agePercent">
 		<item-type>Number</item-type>
```

Comment:
I'm not an expert in UoM, but shouldn't this be Number:Dimensionless?

Reply:
Absolutely. Thanks for the hint. I will change it.